src/Tribe/Importer/Encoding.php

```diff
@@ -0,0 +1,338 @@
+<?php
+/*
+Copyright (c) 2008 Sebasti�n Grignoli
```

Comment:
Yep you'd need to register it via `Tribe__Autoloader::instance()->register_class()` (or `register_prefix()`).